src/index.html

```diff
@@ -11,12 +11,71 @@
       content="ie=edge"
     />
     <title>Moyo header</title>
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
       href="./style.css"
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        class="logo"
+        href="#home"
+      >
+        <img
+          src="./images/logo.png"
+          alt="moyo-logo"
```

Comment:
The `alt` attribute should describe the image content. 'moyo-logo' is not descriptive enough. It's better to describe what the logo represents or looks like, for example, 'Company Moyo Logo'.

Reply:
:)